server/helpers.js

```diff
@@ -119,3 +121,13 @@ export function isCredentialError(err) {
            err.errCode === getErrorCode('INVALID_PARAMETERS') ||
            err.errCode === getErrorCode('NO_PASSWORD');
 }
+
+export function setupMoment(locale) {
+    if (locale) {
```

Comment:
Out of curiosity: it does seem that the `locale` function doesn't return a new value, instead it sets something on the original variable `mom`. So I was wondering whether we really need to `export let moment = mom;`? Does it work if you don't do it?